Add `created_at` and `updated_at` fields (#146)
fivetran/dbt_quickbooks: CHANGELOG.md

# dbt_quickbooks v0.17.0
[PR #146](https://github.com/fivetran/dbt_quickbooks/pull/146) introduces the following updates:

## Feature Updates
- We have introduced the following fields in the `quickbooks__general_ledger` to better analyze real-time transaction data:
  - `created_at`: The time a transaction was first created.
  - `updated_at`: The time a transaction was last updated.
  - This required updates to all our double entry transaction models to bring in these specific fields for each transaction type. 

> - This isn't necessarily a breaking change, but as this introduces new fields to an end model, we determined it was best to bump the version up.

Author: Removed

# dbt_quickbooks v0.17.0
[PR #146](https://github.com/fivetran/dbt_quickbooks/pull/146) introduces the following updates:

## Breaking Changes

> - We have introduced the following fields in the `quickbooks__general_ledger` to better analyze real-time transaction data:

Reviewer: ```suggestion
- Introduced the following fields in the `quickbooks__general_ledger` model to better analyze real-time transaction data:
```

Author: Committed!

  - `created_at`: The time a transaction was first created.
  - `updated_at`: The time a transaction was last updated.

>   - This required updates to all our double entry transaction models to bring in these specific fields for each transaction type.

Reviewer: ```suggestion
  - Updated the `*_double_entry` models to add these fields for each transaction type.
```

Author: Updated.

> - This is a breaking change as this adds new fields to the existing schema for one of our end models. 

Reviewer: ```suggestion
- This is a breaking change as this adds new fields to the existing schema. 
```

Author: Changed.
